atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -144,11 +144,6 @@ public int schemaMutationTimeoutMillis() {
     }
 
     @Value.Default
-    public int heartbeatTimePeriodMillis() {
```

Comment:
Agreed. I think this is a really good idea since it will help educating our users on why we have these things. Also agree that this is probably best done as a separate PR.